packages/tuff-cli-core/src/exporter.ts

```
@@ -284,6 +315,213 @@ function resolveIndexBundleConfig(
+  const { sourcePath, sourceRelativePath } = resolveWidgetSourceFile(context)
+  const source = fs.readFileSync(sourcePath, 'utf-8')
+  const dependencies = ensureVueDependency(validateWidgetDependencies(context.feature.id, source))
+  const widgetId = makeWidgetId(context.pluginName, context.feature.id)
+  const safeId = makeSafeWidgetFileId(widgetId)
+  const compiledRelativePath = posixPath.join('widgets', WIDGET_COMPILED_DIR, `${safeId}.cjs`)
+  const metaRelativePath = posixPath.join('widgets', WIDGET_COMPILED_DIR, `${safeId}.meta.json`)
+  const compiledPath = path.resolve(context.buildDir, compiledRelativePath)
+  const metaPath = path.resolve(context.buildDir, metaRelativePath)
+
+  const descriptor = parse(source, { filename: sourcePath }).descriptor
+  const scriptCode =
```

ESLint

[error] '=' should be placed at the beginning of the line.

(style/operator-linebreak)